sophi.php

```diff
@@ -60,6 +61,7 @@ function() {
 			// Bootstrap.
 			SophiWP\Core\setup();
 			SophiWP\Settings\setup();
+			SophiWP\PostType\post_type();
```

Comment:
To keep consistency, we probably should have a setup() wrapping this function as well @oscarssanchez . What do you think?